### PR TITLE
Added cni plugin

### PR DESCRIPTION
**Pull request by kiranchavala**
Updated the documentation 

The link to install CNI is not valid 

https://projectcalico.docs.tigera.io/manifests/calico.yaml so updated with 

KUBECONFIG=capc-cluster.kubeconfig kubectl apply -f https://raw.githubusercontent.com/weaveworks/weave/master/prog/weave-kube/weave-daemonset-k8s-1.11.yaml

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
@kiranchavala you'll need to signup for the EasyCLA (you may click on the link and agree your contributions as an individual)
/approve

**Comment by weizhouapache** [comment]
@kiranchavala 
Have you tested weavenet ?

I used calico this April, it worked well, except the wrong URL which should be `https://raw.githubusercontent.com/projectcalico/calico/master/manifests/calico.yaml`

**Comment by kiranchavala** [comment]
@weizhouapache tested weave net cni and  its working fine 

Calico is also by me if its pointing to a valid URL

**Comment by kiranchavala** [comment]
@weizhouapache 

Add command for both calico and weave net

**Comment by rohityadavcloud** [comment]
/assign @weizhouapache 
Wei please review and lgtm to merge if you agree with changes, thanks.

File: docs/book/src/development/common.md [hunk 1]
```diff
@@ -60,9 +60,13 @@
...
     ```
 
-6. Install calico on the workload cluster so that pods can see each other
+6. Install calico or weave net cni plugin on the workload cluster so that pods can see each other
     ```
-    KUBECONFIG=capc-cluster.kubeconfig kubectl apply -f https://projectcalico.docs.tigera.io/manifests/calico.yaml
+    KUBECONFIG=capc-cluster.kubeconfig kubectl apply -f https://raw.githubusercontent.com/projectcalico/calico/master/manifests/calico.yaml
+    
+    or 
+
```
Comment (weizhouapache) [inline docs/book/src/development/common.md:68]:
```suggestion
    ```
    or
    ```
```

**Comment by weizhouapache** [comment]
@kiranchavala 
could you sign the CLA ?
image

File: docs/book/src/development/common.md [hunk 2]
```diff
@@ -64,9 +64,16 @@
...
     
-    or 
+    ```    
     
+    ```
+     or
+ 
+    ```
+    
+    ```
```
Comment (weizhouapache) [inline docs/book/src/development/common.md:74]: duplicated lines "```"

File: docs/book/src/development/common.md [hunk 3]
```diff
@@ -108,6 +115,23 @@ kube-system   kube-proxy-6g9zb                                           1/1
...
+kube-system   coredns-5d78c9869d-gphs2                                   1/1     Running     0              21h
+kube-system   etcd-capc-cluster-control-plane-49khm                      1/1     Running     0              21h
+kube-system   kube-apiserver-capc-cluster-control-plane-49khm            1/1     Running     0              21h
+kube-system   kube-controller-manager-capc-cluster-control-plane-49khm   1/1     Running     0              21h
+kube-system   kube-proxy-8lfnm                                           1/1     Running     0              21h
+kube-system   kube-proxy-brj78                                           1/1     Running     0              21h
+kube-system   kube-scheduler-capc-cluster-control-plane-49khm            1/1     Running     0              21h
+kube-system   weave-net-rqckr                                            2/2     Running     1 (3h8m ago)   3h8m
+kube-system   weave-net-rzms4                                            2/2     Running     1 (3h8m ago)   3h8m
+```
```
Comment (weizhouapache) [inline docs/book/src/development/common.md:134]: Great !

File: docs/book/src/development/common.md [hunk 4]
```diff
@@ -60,9 +60,20 @@
...
     ```
-    KUBECONFIG=capc-cluster.kubeconfig kubectl apply -f https://projectcalico.docs.tigera.io/manifests/calico.yaml
+    KUBECONFIG=capc-cluster.kubeconfig kubectl apply -f https://raw.githubusercontent.com/projectcalico/calico/master/manifests/calico.yaml
+    
+    ```    
+    
+    ```
+     or
+ 
+    ```
```
Comment (weizhouapache) [inline docs/book/src/development/common.md:72]:
```suggestion
     or
```

Comment (kiranchavala) [reply docs/book/src/development/common.md]: @weizhouapache    removed the quotes

**Pull request merged**